Assets/Scripts/UI/Core/UISFXHandler.cs

```diff
@@ -12,9 +12,9 @@
 public class UISFXHandler : UIInteractable, IPointerDownHandler
 {
     [SerializeField]
-    string clickEnabledSoundEvent = k.GetPlayEvent(k.MENU_CLICK);
+    public string clickEnabledSoundEvent = k.GetPlayEvent(k.MENU_CLICK);
```

Comment:
These should not need to be `public.` `[SerializeField]` already provides the functionality of letting us edit them in the inspector.

If you need access from a another script, should write a public accessor with a `get` method instead https://msdn.microsoft.com/en-us/library/aa287786(v=vs.71).aspx